libs/external-router/guides/drivers.md

```diff
@@ -0,0 +1,34 @@
+# `@daffodil/external-router` Drivers
+
+The External Router package provides an interface along with a few pre-fabricated drivers for you to simply drop-in to your app and get started with external route resolution.
```

Comment:
"for you simply to drop into"

Reply:
https://github.com/graycoreio/daffodil/pull/1240/commits/e84f901398d30c4af371fb257714725339f3640a

Reply:
https://github.com/graycoreio/daffodil/pull/1240/commits/e84f901398d30c4af371fb257714725339f3640a

Reply:
I don't think "drop-in" is a thing. Maybe "drop in" instead? Though "drop into" makes the most sense to me

Reply:
4bc51801b5c655888e7260fb03f78ca0d2597fab